### PR TITLE
Helm instructions: kubectl apply -> kubectl create

### PR DESCRIPTION
**Pull request by DmitriGekhtman**
Signed-off-by: Dmitri Gekhtman <dmitri.m.gekhtman@gmail.com>

## Why are these changes needed?

Fixes another `apply crd.yaml` to read `create crd.yaml`.
See #271.

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by DmitriGekhtman** [comment]
The nightly compatibility test appears to still be a bit flakey.
On a related note, we should add Ray 2.0.0 to the compatibility matrix.
cc @wilsonwang371

File: helm-chart/kuberay-operator/README.md [hunk 1]
```diff
@@ -18,10 +18,12 @@ version.BuildInfo{Version:"v3.6.2", GitCommit:"ee407bdf364942bcb8e8c665f82e15aa2
 
 To avoid duplicate CRD definitions in this repo, we reuse CRD config in `ray-operator`:
 ```console
-$ kubectl apply -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v0.3.0"
+$ kubectl create -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v0.3.0"
```
Comment (Jeffwan) [inline helm-chart/kuberay-operator/README.md:21]: minor: The PR should be `apply -> create`?

Comment (DmitriGekhtman) [reply helm-chart/kuberay-operator/README.md:21]: oops, fixed

**Comment by wilsonwang371** [comment]
i will add Ray 2.0 in the compatibility test and reorg the tests.

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**